packages/auth/package.json

```diff
@@ -13,25 +15,37 @@
   "bugs": {
     "url": "https://github.com/infra-geo-ouverte/igo2-lib/issues"
   },
+  "scripts": {
+    "build": "ng build auth --configuration production",
+    "publish": "npm publish ../../dist/auth",
+    "test": "ng test auth --watch=false",
+    "watch": "ng build auth --watch"
+  },
+  "exports": {
+    ".": {
+      "import": "./src/public_api.ts"
```

Comment:
Convert all package to an internal package. This will allow to export our raw package via the symlink. This link is possible with the line bellow in the dependency `"@igo2/core": "*"`, this is a workspace feature and will add automatically a link in the node_modules for cross-reference in the workspace. You can inspect your node_modules after a `npm install` to find the symlink in the the @igo2 folder.